### ISSUE TITLE
Calculations for current aligner

### ISSUE THREAD

**Issue by OlenkaP** [RiLights/aligntime#45]
HI @RiLights , this is how we can be calculating current aligner number.

1. current system date + (how many days with each aligner - days have been wearing current aligner for) = new date value

2. once system date reaches that value we get number of current aligner +1 

3. for all the following aligners it would be system date + days on each aligner = new date value2

4. once system reaches that date we would do (current aligner +1) + 1 

for example (today is 13 april, we are wearing aligner number 35, and have been wearing it for 3 days - all of this data is taken from welcome screen set up) : 
1.. 13 April + (7 days - 3 days) = 17 April
2. when the system date is 17th of April, current aligner should change to #36
3. 17 April + 7 days = 24 April
4. On 24th of April current aligner should change to 37, this is all on repeat from now (1st of May should be Aligner 38 and etc)

if we use this formula the only question that would remain is to how we would calculate if number of days for each aligner has been changed through Individual Aligner Adjust screen

**Comment by OlenkaP** [comment]
oh and also 7 days is not hard value its answer to the question how many days you have to wear each aligner for

**Comment by RiLights** [comment]
Where and how you would like to represent this value?

**Comment by OlenkaP** [comment]
I believe, that the best idea would be to have the number here

![111](https://user-images.githubusercontent.com/52235757/79177017-ddfeba80-7e55-11ea-9c04-00a1a65ac3e2.jpg)

as well as representing the right number on the 'modify my treatment' section

**Comment by RiLights** [comment]
Did you check current implementation from build 22?

**Comment by OlenkaP** [comment]
@RiLights , yes, i did, aligner # i see is 73, which is very different to what i expect

**Comment by RiLights** [comment]
Can you please take a look at this test:
https://github.com/RiLights/aligntime/blob/0f29ad86e06d25d7037c358a7b1fee5276a3abad/AlignTimeTests/AlignTimeTests.swift#L640-L660

Is that what you expect? If not, can you please add your values?

**Comment by OlenkaP** [comment]
> @OlenkaP
> Can you please take a look at this test:
> https://github.com/RiLights/aligntime/blob/0f29ad86e06d25d7037c358a7b1fee5276a3abad/AlignTimeTests/AlignTimeTests.swift#L640-L660
> 
> Is that what you expect? If not, can you please add your values?

I'm sorry, I absolutely don't get this test :( 
would you like to talk about it tomorrow?
